### ISSUE TITLE
Add a "copy with to clipboard with forum-markup" to the stats dialog

### ISSUE THREAD

**Issue by stevecotton** [wesnoth/wesnoth#4300]
Posting end-of-scenario stats, as @Konrad22 is doing for his [SotBE reviews](https://r.wesnoth.org/p646805), is a useful addition to the review data. But it would be nicer if it could be in BBcode-formatted text instead of an image.

Feature request: add a copy-to-clipboard button to the stats dialog, that exports as forum-compatible text.

**Comment by jostephd** [comment]
Maybe just copy fixed-width text? That can be pasted on the forum with [code] tags, but can also be pasted on github or discord or anywhere else.

Or maybe we could write a phpbb addon that parses the statistics out of a savefile and displays them?

**Comment by CelticMinstrel** [comment]
Yeah, copying as fixed-width text might be better, and also easier to implement.
